website/docs/maintenance/operations/upgrading.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Upgrading and Compatibility
+sidebar_position: 1
+---
+
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+As an online storage service, Fluss is typically designed to operate over extended periods, often spanning several years.
+Like all long-running services, Fluss requires ongoing maintenance, which includes fixing bugs, implementing improvements,
+and migrating applications to newer versions of the Fluss cluster. In our code design process, we place a strong emphasis
+on compatibility to ensure that any updates or changes are seamlessly integrated without disrupting the existing
+functionality or stability of the system.
+
+This document provides detailed instructions on how to upgrade the Fluss server, as well as information on the
+compatibility between different versions of the Fluss client and the Fluss server.
+
+## Upgrading the Fluss Server Version
+
+This section outlines the general process for upgrading Fluss across versions. For server upgrades, we recommend using
+the rolling upgrade method. Specifically, upgrade the `TabletServers` one-by-one first, and then upgrade the `CoordinatorServer`.
+
+:::note
+1. During the server upgrade process, read and write operations in the cluster will not be affected.
+2. Currently, the Fluss `CoordinatorServer` does not yet support high availability (HA). During the `CoordinatorServer` upgrade stage, the `CoordinatorServer` will be in an unavailable state, which will affect admin operations such as table creation.
+   :::
+
+The following is an example of upgrading the Fluss server from 0.6 to $FLUSS_VERSION$ on
+a [Distributed Cluster](docs/install-deploy/deploying-distributed-cluster.md):
+
+### Download And Configure Fluss
+
+First, download binary file for $FLUSS_VERSION$:
+
+```shell
+tar -xzf fluss-$FLUSS_VERSION$-bin.tgz
+cd fluss-$FLUSS_VERSION$/
+```
+
+Next, copy the configuration options from 0.6 (`fluss-0.6/conf/server.yaml`) to the new configuration
```

Comment:
Before copying configuraing, I think it's better to install optional plugins first, such as add necessary jars required by Streaming Lakehouse feature. 

http://localhost:3000/fluss-docs/docs/maintenance/tiered-storage/lakehouse-storage/#lakehouse-storage-cluster-configurations